primer-api/src/Primer/API.hs

```diff
@@ -1128,6 +1129,7 @@ evalFull' = curry3 $ logAPI (noError EvalFull') $ \(sid, lim, d) ->
             { evalFullReqExpr = e
             , evalFullCxtDir = Chk
             , evalFullMaxSteps = fromMaybe 10 lim
+            , evalFullOptions = UnderBinders -- TODO/REVIEW: should we expose this to the actual api? If so, the BREAKING CHANGE commit message needs updating
```

Comment:
Thoughts? Should the option be exposed to the OpenAPI for EvalFull?

Reply:
Yes. Unless I'm missing something, that's the point of this PR, no?

Reply:
> Yes. Unless I'm missing something, that's the point of this PR, no?

Well, we could always hardcode one particular choice (probably `StopAtBinders`, else this PR is somewhat non-useful). I suppose the question should better be phrased as:

"Should we hardcode `StopAtBinders`, or expose the option to the OpenAPI?"

Reply:
Ahh, I see. There's no harm in exposing it, and it might be helpful at least to work around any bugs we might encounter related to this feature, but the default value should probably be `True`.

Reply:
Agree. I've pushed a new commit exposing it.